### PR TITLE
feat(HOME-3523): update api docs with investigations search and looku…

### PR DESCRIPTION
**Pull request by unjar**
…p routes

### CODE REVIEW

File: api/v10/examples/apexGetInvestigation.res.json [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        "DATE_OF_BIRTH_NOT_VALIDATED":
+        "The date of birth could not be validated against a data source.",
+        "NO_IDENTITY_LOCATED_OR_POOR_MATCH_ON_PRIMARY_DATA_SOURCE":
+        "The identity could not be located or was a poor match on the primary data source."
+      },
+      "results": {
+        "reject": {
+          "NO_IDENTITY_LOCATED_ON_ANY_DATA_SOURCE":
+          "No identity was located on any data source.",
+          "SOCIAL_SECURITY_NUMBER_NOT_VALIDATED": "The SSN is not validated.",
```
Comment (DatChive) [inline api/v10/examples/apexGetInvestigation.res.json]: For consistency's sake, can drop the value down a line

**Comment by DatChive** [comment]
So the V10 partner routes are supposed to be custodian agnostic if I'm not mistaken. Specific routes take a {custodian} var to determine specific behavior.

File: api/v10/examples/apexGetInvestigation.res.json [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    },
+    {
+      "user": "Sketch",
+      "timestamp": "2018-04-04T15:40:51.207",
+      "stateChange": "REJECTED",
+      "comment": "Identity verification failed",
+      "archived": null
+    },
+    {
+      "user": "kevinwong",
```
Comment (bondricek) [inline api/v10/examples/apexGetInvestigation.res.json]: should change to something else

File: api/v10/examples/apexRetrieveInvestigationsList.res.json [hunk 3]
```diff
@@ -0,0 +1,893 @@
...
+    },
+    "created": "2018-04-04T15:40:49.247",
+    "creator": {
+      "subject": "atlas",
+      "userEntity": "apexclearing.local",
+      "userClass": "CLIENT_CREDENTIALS_USER"
+    },
+    "lastUpdated": "2018-05-17T16:41:51.368",
+    "lastUpdater": {
+      "subject": "kevinwong",
```
Comment (bondricek) [inline api/v10/examples/apexRetrieveInvestigationsList.res.json]: same, should remove

**Pull request merged**